README.md

```diff
@@ -63,6 +63,19 @@ This script does the following steps for you:
 - create a TAB (tock application bundle)
 - if you have a J-Link compatible board connected: flash this TAB to your board (using tockloader)
 
+### Enabling rust-embedded support
+
+libtock-rs can be built to be compatible with the rust-embedded
+[embedded_hal](https://docs.rs/embedded-hal/1.0.0/embedded_hal/index.html) by
+including the following when running `make`
```

Comment:
This feels misleading given that the current support just returns errors -- maybe a note that this is a work in progress?

Reply:
The current support allows setting GPIO output state.

I wanted to avoid describing everything that is supported, I have SPI patches ready after this for example. I was hoping the Rust trait system would just indicate what is/isn't supported